src/ansible_navigator/ui_framework/ui.py

```diff
@@ -716,7 +716,7 @@ def _obj_match_filter(self, obj: Dict, columns: List) -> bool:
         :return: True if a match else False
         """
         for key in columns:
-            if self._search_value(self.menu_filter(), obj[key]):
+            if self._search_value(self.menu_filter(), obj.get(key)):
```

Comment:
Looks like this could be in a separate PR.

Reply:
will be.